hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -1139,7 +1139,7 @@ public Properties build() {
         tableConfig.setValue(HoodieTableConfig.BOOTSTRAP_BASE_PATH, bootstrapBasePath);
       }
 
-      if (null != preCombineField) {
+      if (StringUtils.nonEmpty(preCombineField)) {
         tableConfig.setValue(HoodieTableConfig.PRECOMBINE_FIELD, preCombineField);
       }
```

Comment:
`isNullOrEmpty` ?

Reply:
It's actually `isNullOrEmpty` behind `nonEmpty`, do we need call this explicitly? https://github.com/apache/hudi/blob/192ee432ee217d7e388da90a0f6c613ab478ef14/hudi-common/src/main/java/org/apache/hudi/common/util/StringUtils.java#L100-L102

Reply:
No, just confused by this method name.